formats/polus-tabular-to-feather-plugin/README.md

```diff
@@ -0,0 +1,32 @@
+# Tabular to Feather (v0.1.0)
+
+This WIPP plugin does things, some of which involve math and science. There is
+likely a lot of handwaving involved when describing how it works, but handwaving
+should be replaced with a good description. However, someone forgot to edit the
+README, so handwaving will have to do for now. Contact
+[Kelechi Nina Mezu](mailto:nina.mezu@nih.gov) for more
+information.
+
+For more information on WIPP, visit the
+[official WIPP page](https://isg.nist.gov/deepzoomweb/software/wipp).
```

Comment:
The readme needs to be updated to include a description of what the plugin does. A list of supported file formats that we can convert to feather would be helpful.